src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arr = [];
+  let objectFromState = Object.assign({}, state);
+
+  for (let i = 0; i < actions.length; i++) {
+    if (arr.length >= 1) {
```

Comment:
looks strange, do you need it?

Reply:
That was need to take Object after last changes, in 'Fixed' I've tried to apply another method to keep previous changes.